start_ocean.sh

```diff
@@ -269,6 +269,7 @@ while :; do
             eval docker-compose --project-name=$PROJECT_NAME "$COMPOSE_FILES" -f "${NODE_COMPOSE_FILE}" down
             docker network rm ${PROJECT_NAME}_default || true
             docker network rm ${PROJECT_NAME}_backend || true
+	    rm -r ${OCEAN_HOME}
```

Comment:
The artifacts are cleaned out every time the script is run, with or without `--purge`. So if there was some unintended leftover files, could make sense extending the `clean_local_contracts()` function used for this.

But makes sense removing everything, not just artifacts, when purge flag is used. But for this to always work without errors:

- use `-rf`
- check for existence of that folder before trying to remove it, and use quotes + curly braces, like:

```bash
if [ -d "${OCEAN_HOME}" ]; then
  rm -rf "${OCEAN_HOME}"
fi
```